release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md

```diff
@@ -0,0 +1,16 @@
+## 2023-08-31 Version 2.10.0.0
+
+Compatible with OpenSearch-Dashboards 2.10.0
+
+### Enhancements
+* Security Getting Started page follows Dashboard themes ([#1538](https://github.com/opensearch-project/security-dashboards-plugin/pull/1538))
+* Security Roles and Audit Log Settings follow Dashboard themes ([#1558](https://github.com/opensearch-project/security-dashboards-plugin/pull/1558))
+* Support OpenSearch logo theming for light / dark modes ([#1568](https://github.com/opensearch-project/security-dashboards-plugin/pull/1568))
```

Comment:
should we also include #1550 and #1559 here?

https://github.com/opensearch-project/security-dashboards-plugin/commits/2.10

Reply:
nvm. I see the original PRs were included here.